q2_moshpit/busco/busco.py

```diff
@@ -159,54 +215,32 @@ def _visualize_busco(output_dir: str, busco_results: pd.DataFrame) -> None:
             label_font_size=17,
             spacing=20
         )
-        context.update(
-            {f"sample{i}": {
+        context.update({
+            f"partition_{i}":
+            {
                 "subcontext": subcontext,
-                "sample_counter": sample_counter,
-                "sample_ids": df['sample_id'].unique().tolist(),
-            }}
-        )
-
-    marker_summary_spec = _draw_marker_summary_histograms(busco_results)
-    selectable_summary_spec = _draw_selectable_summary_histograms(
-        busco_results
-    )
+                "counters": counters,
+                "ids": df[column_name].unique().tolist(),
+            }
+        })
 
+    # Render
     vega_json = json.dumps(context)
-    vega_json_summary = json.dumps(marker_summary_spec)
-    vega_json_summary_selectable = json.dumps(selectable_summary_spec)
-
-    # Copy BUSCO results from tmp dir to output_dir
-    TEMPLATES = os.path.join(
-        os.path.dirname(os.path.dirname(__file__)), "assets"
-    )
-    index = os.path.join(TEMPLATES, "busco", "index.html")
-    details = os.path.join(TEMPLATES, "busco", "detailed_view.html")
-    table = os.path.join(TEMPLATES, "busco", "table.html")
-    copytree(
-        src=os.path.join(TEMPLATES, "busco"),
-        dst=output_dir,
-        dirs_exist_ok=True
-    )
-
+    vega_json_sum = json.dumps(_draw_marker_summary_histograms(busco_results))
```

Comment:
😅 guilty! the line was too long with the old name so I made it shorter so it would fit in one line. I only had aesthetic reasons for this. I can change it back if you'd like.

Reply:
Please do - you can just break the line ;)